go.sum

```diff
@@ -11,6 +11,10 @@ github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c
 github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
 github.com/dynatrace/dynatrace-configuration-as-code-core v0.7.1-0.20241120133258-d514d28440ee h1:hsGmVOg0uv7FENo3PIbrbnxM/0A0XPr3vf/sBtdhuOg=
 github.com/dynatrace/dynatrace-configuration-as-code-core v0.7.1-0.20241120133258-d514d28440ee/go.mod h1:ZEqPmEOAnMtCzPmIJ4Rd9nkX6qmLup9Az0r1e1KlIeU=
+github.com/dynatrace/dynatrace-configuration-as-code-core v0.7.1-0.20241202121206-ae7ea8677c89 h1:vhkYF5nMjEEHzeFeE5ZvdKzhLVy6mK/pzNAWO4YOI5A=
```

Comment:
go mod tidy

Reply:
I am wondering why our pipeline is not failing. Should be 🤔